OrbitClientData/FunctionUtils.cpp

```diff
@@ -50,13 +50,16 @@ std::unique_ptr<FunctionInfo> CreateFunctionInfo(const SymbolInfo& symbol_info,
   return function_info;
 }
 
+#define STUB_PARAMS \
+  "(unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long)"
```

Comment:
Cant this be solved without a macro? 
I feel less macros is better than more macros.. but I am unsure

Reply:
Yeah I agree, although concatenating string literals happen at compile time... I did change it to make the code cleaner.